nussknacker-dist/src/universal/bin/run.sh

```diff
@@ -3,12 +3,42 @@
 #TODO handle JAVA_OPTS etc. - maybe use sbt-native-packager scripts generator?
 
 NUSSKNACKER_DIR=`dirname "$0" | xargs -I{} readlink -f {}/..`
-LIB_DIR="$NUSSKNACKER_DIR/lib"
+export STORAGE_DIR="$NUSSKNACKER_DIR/storage"
 CONF_DIR="$NUSSKNACKER_DIR/conf"
+LIB_DIR="$NUSSKNACKER_DIR/lib"
+
 LOGS_DIR="$NUSSKNACKER_DIR/logs"
 LOG_FILE="$LOGS_DIR/frontend.log"
 PID_FILE="$NUSSKNACKER_DIR/frontend.pid"
 
+if [[ "${USE_DOCKER_ENV}" == "true" ]]; then
```

Comment:
I think we define defaults on too many levels, why don't we have ENV-or-default in app.conf, 
and override them only in docker?
I think this run is already too large and we should try to use sth default

Reply:
We already have default on app.conf level, see here:
```
flinkRestUrl: "http://jobmanager:8081"
flinkRestUrl: ${?FLINK_REST_URL}
```
Those defaults are about profiles that will be hard to manage in lightbend config (also will complicate them). I've lightened it a bit (removed some auth conf) but can't see options to lightened it more.

You ask me to extract this logic to separate file. I can do this, but it will be rather extraction of this demonizing logic to e.g. run-demonized.sh, because env vars should stick to execution of process. WDYT? Personally I would like to do this in other PR with transition to e.g. java service wrapper...